modules/tests/shared/src/test/scala/gsp/math/AngleSpec.scala

```diff
@@ -135,4 +134,29 @@ final class AngleSpec extends CatsSuite {
     }
   }
 
+  test("Difference with itself is 0") {
+    forAll { (a: Angle) =>
+      a.difference(a) shouldEqual Angle.Angle0
+    }
+  }
+
+  test("Difference with reverse is 180") {
+    forAll { (a: Angle) =>
+      a.difference(a + Angle.Angle180) shouldEqual Angle.Angle180
+    }
+  }
+
+  test("Difference is in the range [0 .. π)") {
```

Comment:
[0 .. π] … this test should fail for (0, 180)

Reply:
Are you using `[]` for an inclusive range, i.e `0 <= x <= pi`?

Reply:
Yes, `[` for inclusive, `(` for exclusive.